cloud-regionsrv-client.spec

```diff
@@ -110,6 +109,9 @@ cp -r usr %{buildroot}
 python3 setup.py install --prefix=%{_prefix}  --root=%{buildroot}
 mkdir -p %{buildroot}/var/lib/regionService/certs
 mkdir -p %{buildroot}/var/lib/cloudregister
+install -d -m 755 %{buildroot}/%{_mandir}/man1
+install -m 644 man/man1/* %{buildroot}/%{_mandir}/man1
+gzip %{buildroot}/%{_mandir}/man1/*
```

Comment:
Is it redundant to call `gzip` on man contents in both the specfile and the makefile?

Re: https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/26/files#diff-b67911656ef5d18c4ae36cb6741b7965R22